bindings/python/src/jls.rs

```diff
@@ -0,0 +1,38 @@
+use pyo3::prelude::*;
+use pyo3::{PyIterProtocol, PyObjectWithToken};
+
+use jail as native;
+use running::RunningJail;
+
+#[pyclass]
+pub struct Jls {
+    token: PyToken,
+    it: native::RunningJailIter,
```

Comment:
maybe call this `iter` for clarity

Reply:
hmm, wasn't sure whether that was a good idea since it implements Iter, but yeah, why not?